hack/make-rules/verify.mk

```diff
@@ -38,43 +38,25 @@ CODE_MAINT += tidy
 tidy:
 	go mod tidy
 
-GOLINT_LINTERS ?= \
-	--disable-all \
-	--enable=deadcode \
-	--enable=dogsled \
-	--enable=dupl \
-	--enable=errcheck \
-	--enable=gocritic \
-	--enable=gofmt \
-	--enable=revive \
-	--enable=gosimple \
-	--enable=govet \
-	--enable=ineffassign \
-	--enable=misspell \
-	--enable=nakedret \
-	--enable=structcheck \
-	--enable=typecheck \
-	--enable=unconvert \
-	--enable=unused \
-	--enable=varcheck \
-	--enable=whitespace
-
-CODE_MAINT += revive
 .PHONY: revive
 revive: $(TOOLBIN)/revive
 	$(TOOLBIN)/revive -config lint-rules.toml -formatter stylish ./...
 
-CODE_MAINT += lint
+.PHONY: lint
 lint: $(TOOLBIN)/golangci-lint
-	$(TOOLBIN)/golangci-lint run ${GOLINT_LINTERS} --timeout=5m ./...
+	$(TOOLBIN)/golangci-lint run
 
 .PHONY: lint-fix
 lint-fix: $(TOOLBIN)/golangci-lint
-	$(TOOLBIN)/golangci-lint run --fix ${GOLINT_LINTERS} ./...
+	$(TOOLBIN)/golangci-lint run --fix
 
-.PHONY: lint-todo
-lint-todo: $(TOOLBIN)/golangci-lint
-	$(TOOLBIN)/golangci-lint run --enable=godox ${GOLINT_LINTERS} ./...
+.PHONY: lint-updated
+lint-updated: $(TOOLBIN)/golangci-lint
+	$(TOOLBIN)/golangci-lint run --new-from-rev=HEAD~
+
+.PHONY: lint-godox
+lint-godox: $(TOOLBIN)/golangci-lint
+	$(TOOLBIN)/golangci-lint run --enable=godox
```

Comment:
Why is godox disabled in main config? Why does it need its own target? Also, godox is opiniated and we do use TODO comments in out code, like many other projects. You can change that of course :-)

Reply:
it was one of my questions to you ;-)